### DISCUSSION TITLE
WDIO on Ms Edge - No Alert Issue

### DISCUSSION THREAD

**Discussion by devqaio** [webdriverio/webdriverio#13845] [General]
I am trying to execute a cucumber feature on MS Edge. I have noticed, that for every command, it looks for an alert to interact. I am not new to Selenium but am completely new to webdriverio. Is this expected due to any config issues in my wdio.conf as below?

`export const config: WebdriverIO.Config = {
  runner: 'local',
  tsConfigPath: './tsconfig.json',

  specs: ['./test/features/**/*.feature'],
  exclude: [
    // 'path/to/excluded/files'
  ],
  maxInstances: 10,
  capabilities: [
    {
      browserName: 'MicrosoftEdge',
      'wdio:maxInstances': 5,
      acceptInsecureCerts: true,
      'wdio:enforceWebDriverClassic': true,
      'ms:edgeOptions': {
        args: [
          '--disable-notifications',
          '--disable-popup-blocking',
          '--disable-infobars',
          '--start-maximized'
        ],
      },
    },
  ],

  logLevel: 'warn',
  bail: 0,
  waitforTimeout: 10000,
  connectionRetryTimeout: 120000,
  connectionRetryCount: 3,
  services: [
    'visual',
    'cucumber-viewport-logger',
    [
      'rerun',
      {
        rerunDataDir: './reports/rerun',
      },
    ],
  ],

  framework: 'cucumber',
  reporters: [
    'spec',
    'dot',
    [
      'junit',
      {
        outputDir: './reports/junit-reports',
        outputFileFormat: function (options) {
          // optional
          return `results-${options.cid}.${options.capabilities}.xml`;
        },
      },
    ],
    [
      'json',
      {
        outputDir: './reports/json-results',
      },
    ],
    [
      'video',
      {
        outputDir: './reports/video-results',
      },
    ],
    [
      'allure',
      {
        outputDir: './reports/allure-results',
        disableWebdriverStepsReporting: false,
        disableWebdriverScreenshotsReporting: false,
      },
    ],
    [
      'html-nice',
      {
        outputDir: './reports/html-reports/',
        filename: 'report.html',
        reportTitle: 'Test Report Title',
        linkScreenshots: true,
        //to show the report in a browser when done
        showInBrowser: true,
        collapseTests: false,
      },
    ],
    [
      'cucumberjs-json',
      {
        jsonFolder: './reports/cucumber-json',
        language: 'en',
      },
    ],
    [
      'timeline',
      {
        outputDir: './reports/timeline-report',
      },
    ],
  ],

  cucumberOpts: {
    require: ['./test/step-definitions/*.ts'],
    backtrace: false,
    requireModule: [],
    dryRun: false,
    failFast: false,
    name: [],
    snippets: true,
    source: true,
    strict: false,
    tagExpression: '',
    timeout: 60000,
    format: ['pretty'],
    profiles: [],
  },

  beforeCommand: function (commandName, args) {
    console.log(`Command: ${commandName}`, args);
  },

  afterCommand: function (commandName, args, result, error) {
    console.log(`Finished command: ${commandName}`, args);
    if (error) console.error('Error:', error);
  },
};
`

**Comment by christian-bromann** [comment]
@gitissuepost thanks for raising this discussion, mind providing a minimal reproducible example?

**Reply by devqaio** [reply]
@christian-bromann - Attached is my sample project with a basic feature file around google search. Moreover, attached the console log (trace loglevel) generated at my end. I am using Windows 11. Observes the same with Chrome as well

[wdio.log](https://github.com/user-attachments/files/17611793/wdio.log)
[wdio-sample.zip](https://github.com/user-attachments/files/17611802/wdio-sample.zip)

**Reply by devqaio** [reply]
@christian-bromann - Seems the issue is with cucumber-viewport-logger service. If I remove that from list, it works without issue. Not sure if this need to go to github issues of that service as I couldn't find any documentation to handle that and the repo is not being maintained since few years.

**Reply by christian-bromann** [reply]
> the repo is not being maintained since few years.

This is usually a bad sign, likely that the project is not compatible with your WebdriverIO version.

**Reply by christian-bromann** [reply]
I have raised [an issue](https://github.com/viktor-silakov/wdio-cucumber-viewport-logger-service/issues/2) and requested to have it moved to the WDIO org so we can continue to maintain it

**Reply by devqaio** [reply]
@christian-bromann - Thank you for this.. 

Does the latest wdio version supports video reporting? I had configured the way it has been recommended, but it doesn't generate the video file. The reporter log shows a video file path but that never gets created.

**Reply by devqaio** [reply]
Also, I see a weird issue in npmjs (https://www.npmjs.com/package/wdio-cucumber-viewport-logger-service). The home page for this plugin is pointing to a different github repo. 

As per package.json from official repo, the published version was 1.0.4 but when I use npm install command, it downloads 2.1.36-alpha.0 which is published version of syngrisi pkg. Not sure if the capabilities of that repo is creating conflicts.

**Reply by christian-bromann** [reply]
> Does the latest wdio version supports video reporting?

WebdriverIO maintains a [video reporter](https://www.npmjs.com/package/wdio-video-reporter).



> Also, I see a weird issue in npmjs ([npmjs.com/package/wdio-cucumber-viewport-logger-service](https://www.npmjs.com/package/wdio-cucumber-viewport-logger-service)). The home page for this plugin is pointing to a different github repo.

This is indeed weird, I guess I filed the issue at a wrong place then. I can ask in the issue.

**Reply by christian-bromann** [reply]
Update: it seems like the package is just an export of the [original location](https://github.com/syngrisi/syngrisi) which is a monorepo. The project author contributes to both packages and seems to be the original author of it.
